website_base_multi_image/models/image.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import base64
+from openerp import api, models
+
+
+class Image(models.Model):
+    _inherit = "base_multi_image.image"
+
+    @api.model
+    def get_images_to_add(self, post, model_name):
+        """
+        @param post: dict with images in format 'new_image_xxx'
```

Comment:
`'new_image_xxx'` is not a dict format nor an image format, I don't understand this doc line. Besides, add a general description for what the method does in its 1st line please.